src/components/Header/index.module.scss

```diff
@@ -277,4 +289,4 @@ $defaultHeaderHeight: 90px;
       cursor: unset;
     }
   }
-}
+}
```

Comment:
```suggestion
}

```
![image](https://github.com/Magickbase/nervos-official-website/assets/7271329/6f3a1802-624d-4c8b-97ef-5562184f2004)
A newline is required at the end

Reply:
Added new line and eslint/stylelint rule to enforce this: https://github.com/zhangyouxin/nervos-official-website/commit/c607255fb65712145d9afc9a9c6fb655dbf8e448

Reply:
> Added new line and eslint/stylelint rule to enforce this: [zhangyouxin@c607255](https://github.com/zhangyouxin/nervos-official-website/commit/c607255fb65712145d9afc9a9c6fb655dbf8e448)

Strange, prettier should take care of this formatting, no need for eslint. Is it that prettier is not being run automatically on your side, or it is being run but our formatting settings are different?